code/src/WijmoProvider/Helper/DataUtils.ts

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace WijmoProvider.Helper.DataUtils {
+    export function GetTicksFromDate(
+        value: Date,
+        hasTimeZone: boolean
+    ): string {
+        let ticks;
+
+        if (hasTimeZone) {
+            ticks = value.getTime();
+        } else {
+            ticks = value.getTime() - value.getTimezoneOffset() * 60 * 1000;
+        }
+
+        return ticks;
+    }
+
+    export function TrimSeconds(value: string): string {
```

Comment:
What is the difference between this method and the "trimSecondsFromDate" previously defined?

Reply:
The TrimSecondsForDate will trim the seconds: 2021-06-25 12:23:34 => 2021-06-25 12:23 -> This is required format for SetCellData from wijmo

The TrimSeconds will clean the seconds: 2021-06-25T12:23:34Z => 2021-06-25T12:23:00Z -> This is required for comparisons in Conditional Format - Will rename this!